assets/style.scss

```diff
@@ -290,4 +290,8 @@ green     #859900  operators, other keywords
 .highlight .vc { color: #268bd2 } /* Name.Variable.Class */
 .highlight .vg { color: #268bd2 } /* Name.Variable.Global */
 .highlight .vi { color: #268bd2 } /* Name.Variable.Instance */
-.highlight .il { color: #2aa198 } /* Literal.Number.Integer.Long */
\ No newline at end of file
+.highlight .il { color: #2aa198 } /* Literal.Number.Integer.Long */
+
+.silent-link{
```

Comment:
tachyons has `.link` for this

Reply:
Still leaves an underline without the "!important" because of other styles. Now merging.